src/v/redpanda/application.cc

```diff
@@ -429,7 +429,7 @@ static storage::kvstore_config kvstore_config_from_global_config() {
      */
     return storage::kvstore_config(
       config::shard_local_cfg().kvstore_max_segment_size(),
-      config::shard_local_cfg().kvstore_flush_interval(),
+      config::shard_local_cfg().kvstore_flush_interval.bind(),
```

Comment:
does this return a `config::binding` that is "wired-up" for dynamically being changed?

Reply:
Yep.